feat(workflows): add helper and workflow to keep keystone and nautobot in sync (#242)
rackerlabs/understack: docs/component-understack-workflows.md

# Understack Workflows

This is the Kubernetes installation of the Argo Workflows
and their associated support bits to add the actual workflows,
sensors and triggers into a Kubernetes cluster.

Due to the scoping of resources into different namespaces, this
must also be split into multiple namespaces.

This code lives in `apps/understack-workflows` of this repo.

Specifics about the workflows can be seen in the Workflows
section.

## eventsource-openstack

The resources managed here are:

> 1. A RabbitMQ user on the OpenStack RabbitMQ cluster
> named `argo` currently and gives it permissions to listen
> to notifications from OpenStack components. At this time it

Reviewer: ```suggestion
1. A RabbitMQ user named `argo` on the OpenStack RabbitMQ cluster, which has permissions to listen for notifications from OpenStack components. At this time it
```
